README.md

```diff
@@ -51,9 +52,74 @@ describe('MyWidget', () => {
 
 ## Vitest
 
-This package has an optional peer dependency on `vitest`. Any vitest-specific
-utility is exposed in a separate `@hypothesis/frontend-testing/vitest` entry
-point.
+All projects depending on this package are expected to use Vitest to run tests.
+
+However, we don't follow all practices recommended by Vitest. You should take
+these points into consideration when reading Vitest's documentation:
+
+1. By default, Vitest uses [Vite](https://vite.dev/) to bundle the tests and
+    source code as test files are run.
+    Instead, we manually pre-bundle our tests and source code in a single file
+    with Rollup, and that's the only test file we run.
+    That file is still processed by Vite, but since it has nothing to transform,
+    the time it takes is negligible.
```

Comment:
There is one extra indentation space here which is unnecessary. You can reformat the file using `prettier -w README.md`.